app/views/posts/index.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<h1>Here is a lists of a given user</h1>
+<section>
+    <div class= "container">
+        <div class="user-img">
+          <img src="<%= @user.photo %>" alt="<%= @user.name %>">
+        </div>
+        <div class="details">
+          <h1 class="user-head"><%= @user.name %></h1>
+          <p class="user-para">Number of posts: <%= @user.posts_counter || 0 %></p>
+    </div>
+  <% @user.recent_posts.each do |post| %>
+    <%= link_to user_post_path(@user, post) do %>
+      <h2>Post: <%= post.id %></h2>
+      <% end %>
+      <div class= "container">
+        <div class="details">
+          <h1 class="user-head"><%= post.title %></h1>
+          <p>post description: <span><%= post.text%></span> </p>
```

Comment:
* Kindly solve N+1 problem when fetching all posts and their comments for a user.  for example while fetching all posts for a user you can include comments as well `@posts = @user.posts.includes(:comments)`